libs/designsystem/card/src/card.component.spec.ts

```diff
@@ -45,36 +49,75 @@ describe('CardComponent', () => {
     });
   });
 
-  describe('with mode attribute', () => {
+  describe('with flat variant attribute', () => {
+    let cardElement: HTMLElement;
```

Comment:
Consider declaring this top level like we do with spectator to avoid repitition 🔁 😄